client/src/components/Draftail/CommentableEditor/CommentableEditor.js

```diff
@@ -0,0 +1,217 @@
+import { DraftailEditor, ToolbarButton, createEditorStateFromRaw, serialiseEditorStateToRaw } from 'draftail';
```

Comment:
Now that we’re using TypeScript in Wagtail, wouldn’t you want to write this file in TypeScript?

Reply:
Probably a good idea! Was trying to keep with the rest of the Draftail stuff for now, but it's probably best just to switch

Reply:
Plan is now to do this in a future ticket, so resolving this for now